common/types/loadbalancer.go

```diff
@@ -57,17 +83,30 @@ type RevNATMap map[ServiceID]L3n4Addr
 type LoadBalancer struct {
 	BPFMapMU  sync.RWMutex
 	SVCMap    SVCMap
+	SVCMapID  SVCMapID
```

Comment:
Feel free to fix it up afterwards. We have to limit the scope of this PR somewhere. There are several issues that need to be addressed in this regard.